capnslog/README.md

```diff
@@ -0,0 +1,41 @@
+# CoreOS Log
+
+There are far too many logging packages out there, with varying degrees of licenses, far too many features (colorization, all sorts of log frameworks) or are just a pain to use (lack of `Fatalln()`?)
+
+## Design Principles
+
+* Logging is off by default, except critical failures, which are unrecoverable.
```

Comment:
It's only an issue when things log and you have *no* control. We're building those controls right now, so it should not be an issue. Let's log at INFO